app/views/reports/admin_dashboard.html.erb

```diff
@@ -0,0 +1,51 @@
+<html lang="<%= @locale %>">
+<% title t('admin.dashboard') %>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+</head>
+
+<body>
+
+<div class="container">
+  <h1>Reports Table for Admin </h1>
+  <table class="table table-hover" border=1px >
```

Comment:
This is fine for now, but ideally, we should be using flex box for tables. We can migrate that in another PR.